docs/cold_reset.md

```diff
@@ -0,0 +1,113 @@
+# Performing cold reset remotely
+
+It is not possible to perform 100% cold boot without physically disconnecting
+power supply, some registers cannot be reset otherwise.
+[BKDG](https://www.amd.com/system/files/TechDocs/52740_16h_Models_30h-3Fh_BKDG.pdf)
+differentiates between cold boot, cold reset and warm reset, but even within
+one type of reset some bits marked with "Cold reset: 0" are cleared and others
+are not.
+
+### Discovered ways to perform reset
+
+| Reset type       | D18F0x6C   | PMxC0      |
+|------------------|------------|------------|
+| Cold boot        | 0x000ff800 | 0x00000800 |
+| Reboot           | 0x000ffe00 | 0x40080400 |
+| Reset            | 0x000ff800 | 0x40010400 |
+| Power button     | 0x000ff800 | 0x40200402 |
+| FullRst          | 0x000ff800 | 0x40200400 |
+| ACPI reset       | 0x000ff800 | 0x40000400 |
+| PCI reset        | 0x000ffe00 | 0x40020400 |
+
+These are initial values, they are changed during boot sequence. Bits changing
+in D18F0x6C are checked by AGESA to differentiate cold boot from warm boot.
```

Comment:
@krystian-hebel provide at least register names if available. Raw numbers do not say much about their purpose

Reply:
done